src/lib/validations/MobilePrimitives_NumericResponseBlockValidator.ts

```diff
@@ -0,0 +1,68 @@
+import {IBlock, IBlockConfig, INumericBlockConfig} from '@floip/flow-runner'
+import {ExtendedValidatorBase, ValidationResults} from './ExtendedValidatorBase'
+
+function normalizeNumericConfigProperty(value?: number | string | null): [number, boolean] {
+  let normalizedValue = 0
+  let hasValue = false
+
+  if (value !== undefined && value !== null) {
+    hasValue = true
+    normalizedValue = typeof value === 'number' ? value : parseInt(value, 10)
+  }
+
+  return [
+    normalizedValue,
+    hasValue,
+  ]
+}
+
+export default class MobilePrimitives_NumericResponseBlockValidator extends ExtendedValidatorBase {
+  static runExtraValidations(block: IBlock<IBlockConfig>): ValidationResults | null {
```

Comment:
This code has been copied from `store/flow/block-types/MobilePrimitives_NumericResponseBlockStore` with no substantial change.

The only difference is using `.push` instead of `SupplementaryAJVErrorsBuilder` 😉